packages/generator/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "@thingweb/ui-wot-generator",
+  "version": "0.1.0",
+  "description": "A library capable of creating ui components from a W3C Thing Description",
```

```suggestion
  "description": "A library capable of creating UI components from a W3C Thing Description",
```
adding @egekorkan 's feedback